osu.Desktop/OsuGameDesktop.cs

```diff
@@ -59,7 +59,7 @@ private string getStableInstallPath()
             try
             {
                 using (RegistryKey key = Registry.ClassesRoot.OpenSubKey("osu"))
-                    stableInstallPath = key?.OpenSubKey(@"shell\open\command")?.GetValue(string.Empty).ToString().Split('"')[1].Replace("osu!.exe", "");
+                    stableInstallPath = key?.OpenSubKey(@"shell\open\command")?.GetValue(string.Empty).ToString()?.Split('"')[1].Replace("osu!.exe", "");
```

Comment:
This nullable warning is quite unprecise.
`GetValue` declares to return nullable, but no warning (I don't really expect it because we haven't fully opt-in NRT).

Here, `object.ToString()` is declared nullable, but seldom nullable in override in practice. I reported an issue for them always treating `ToString()` non-nullable, but they look to aggressive now.

Reply:
/shrug